rpm/sailfish-browser.spec

```diff
@@ -101,17 +101,20 @@ Unit tests and additional data needed for functional tests
 %install
 rm -rf %{buildroot}
 %qmake5_install
-chmod +x %{buildroot}/%{_oneshotdir}/*
+chmod +x %{buildroot}/%{_oneshotdir}/* || :
```

Comment:
Don't think we need || : protection on compile time, i.e. on %install. Better fail if packaging doesn't succeed.

Reply:
Yes, that is correct. Shouldn't be used in %install.